strawberry/lazy_type.py

```diff
@@ -78,8 +78,16 @@ def __init__(self, module: str) -> None:
             assert frame is not None
             self.package = cast(str, frame.f_globals["__package__"])
 
-    def resolve_forward_ref(self, forward_ref: ForwardRef) -> LazyType:
-        return LazyType(forward_ref.__forward_arg__, self.module, self.package)
+    def resolve_forward_ref(self, forward_ref: Union[ForwardRef, str]) -> LazyType:
+        return LazyType(
+            (
+                forward_ref.__forward_arg__
+                if isinstance(forward_ref, ForwardRef)
+                else forward_ref
+            ),
+            self.module,
+            self.package,
+        )
```

Comment:
same as above, when would you receive a str here? and why it is ok to pass a string? 😊

Reply:
Oh, totally here! Also something that I changed when starting the refactoring of the annotation class, but it is not required based on how the callsites work anymore. Reverting this!